spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/util/ObservableReactiveUtil.java

```diff
@@ -68,6 +72,21 @@ private ObservableReactiveUtil() {
 		});
 	}
 
+	public static <ResponseT, RequestT, A> Flux<ResponseT> streamingBidirectionalCall(
+			Function<StreamObserver<ResponseT>, StreamObserver<RequestT>> remoteCall,
+			Flux<A> inputs, BiFunction<A, Flux<ResponseT>, Mono<RequestT>> requestFunc) {
```

Comment:
I'd rename `inputs` to `entitiesToSave` or something else that more precisely reflects the purpose.